cumulus/parachains/runtimes/assets/asset-hub-westend/src/bridge_to_rococo_config.rs

```diff
@@ -0,0 +1,311 @@
+	>;
+	// Converter aligned with `OpenBridgeOrigin`.
+	type BridgeOriginAccountIdConverter =
+		(ParentIsPreset<AccountId>, SiblingParachainConvertsVia<Sibling, AccountId>);
+
+	type BridgeDeposit = BridgeDeposit;
+	type Currency = Balances;
+	type RuntimeHoldReason = RuntimeHoldReason;
+	// Do not require deposit from system parachains (including itself) or relay chain
+	type AllowWithoutBridgeDeposit =
+		(RelayOrOtherSystemParachains<AllSiblingSystemParachains, Runtime>, Equals<HereLocation>);
+
+	// This pallet is deployed on AH, so we expect a remote router with `ExportMessage`. We handle
+	// congestion with XCM using `udpate_bridge_status` sent to the sending chain. (congestion with
+	// local sending chain)
+	type LocalXcmChannelManager = HereOrLocalConsensusXcmChannelManager<
+		pallet_xcm_bridge::BridgeId,
+		// handles congestion for local chain router for local AH's bridges
+		ToRococoOverAssetHubRococoXcmRouter,
+		// handles congestion for other local chains with XCM using `update_bridge_status` sent to
+		// the sending chain.
+		UpdateBridgeStatusXcmChannelManager<
+			Runtime,
+			XcmOverAssetHubRococoInstance,
+			UpdateBridgeStatusXcmProvider,
+			xcm_config::LocalXcmRouter,
+		>,
+	>;
+	// Dispatching inbound messages from the bridge and managing congestion with the local
+	// receiving/destination chain
+	type BlobDispatcher = BlobDispatcherWithChannelStatus<
+		// Dispatches received XCM messages from other bridge
+		BridgeBlobDispatcher<
+			// TODO: FAIL-CI wait for https://github.com/paritytech/polkadot-sdk/pull/6002#issuecomment-2469892343
+			xcm_config::LocalXcmRouter,
+			UniversalLocation,
+			(),
+		>,
+		// Provides the status of the XCMP queue's outbound queue, indicating whether messages can
+		// be dispatched to the sibling.
+		cumulus_pallet_xcmp_queue::bridging::OutXcmpChannelStatusProvider<Runtime>,
+	>;
+	type CongestionLimits = ();
+}
+
+/// XCM router instance to the local `pallet_xcm_bridge_hub::<XcmOverAssetHubRococoInstance>` with
+/// direct bridging capabilities for `Rococo` global consensus with dynamic fees and back-pressure.
+pub type ToRococoOverAssetHubRococoXcmRouterInstance = pallet_xcm_bridge_router::Instance4;
+impl pallet_xcm_bridge_router::Config<ToRococoOverAssetHubRococoXcmRouterInstance> for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type WeightInfo =
+		weights::pallet_xcm_bridge_router_to_rococo_over_asset_hub_rococo::WeightInfo<Runtime>;
+
+	type DestinationVersion = PolkadotXcm;
+
+	// We use `LocalExporter` with `ViaLocalBridgeHubExporter` ensures that
+	// `pallet_xcm_bridge_router` can trigger directly `pallet_xcm_bridge` as exporter.
+	type MessageExporter = pallet_xcm_bridge_router::impls::ViaLocalBridgeHubExporter<
+		Runtime,
+		ToRococoOverAssetHubRococoXcmRouterInstance,
+		LocalExporter<XcmOverAssetHubRococo, UniversalLocation>,
+	>;
+
+	// For congestion - resolves `BridgeId` using the same algorithm as `pallet_xcm_bridge_hub` on
```

Comment:
please change the same for AHR

```suggestion
	// For congestion - resolves `BridgeId` using the same algorithm as `pallet_xcm_bridge` on
```